src/pynamodb_utils/exceptions.py

```diff
@@ -10,3 +10,15 @@ class FilterError(Error):
 
 class SerializerError(Error):
     pass
+
+
+class IndexNotFoundError(Exception):
+    def __init__(self, message: str):
+        self.message = message
```

Comment:
@bmlnr Do we use message attributes anywhere?

Reply:
I created this exception to be thrown instead of the overly generic Exception. Originally, it had a message in it which I left intact. In any case, it is good that it contains a message (during debugging etc.).

Reply:
I wonder if `message` is redundant here. You could always access exception message like in the example below:

```python
class IndexNotFoundError(Exception):
    pass

ex = IndexNotFoundError("This is message")
print(ex.args[0])
```

Reply:
What you think? Do we still need this message attr?